exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -217,6 +218,8 @@ private ServletContextHandler createServletContextHandler(final boolean authEnab
       //DrillSecurityHandler is used to support SPNEGO and FORM authentication together
       servletContextHandler.setSecurityHandler(new DrillHttpSecurityHandlerProvider(config, workManager.getContext()));
       servletContextHandler.setSessionHandler(createSessionHandler(servletContextHandler.getSecurityHandler()));
+      SessionCookieConfig sessionCookieConfig = servletContextHandler.getServletContext().getSessionCookieConfig();
+      sessionCookieConfig.setHttpOnly(true);
```

Comment:
I would recommend to move this logic inside `createSessionHandler()` as below:

`sessionManager.getSessionCookieConfig().setHttpOnly(true);`